docs/reference/config-params.md

```diff
@@ -203,75 +234,88 @@ Otherwise, number of replicas will default to `1`.
 
 ### Possible options: Arbitrary integer value. Example: `10`.
 
-> replicas:
+> x-k8s.workload.replicas:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.replicas: 3
+    x-k8s:
+      workload:
+        replicas: 1
 ...
 ```
 
-## kev.workload.autoscale-max-replicas
+## x-k8s.workload.autoscale
+
+Configures an application for auto-scaling. See K8s [documentation](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+
+### x-k8s.workload.autoscale.maxReplicas
 
 Defines the maximum number of instances (replicas) the application component should automatically scale up to. See K8s [documentation](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/). This setting is only taken into account when initial number of replicas is lower than this parameter.
 
-### Default: `0`
+#### Default: `0`
 
-### Possible options: Arbitrary integer value. Example: `10`.
+#### Possible options: Arbitrary integer value. Example: `10`.
 
-> autoscale-max-replicas:
+> x-k8s.workload.autoscale.maxReplicas:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.autoscale-max-replicas: 3
+    x-k8s:
+      workload:
+        autoscale:
+          maxReplicas: 10
 ...
 ```
 
-## kev.workload.autoscale-cpu-threshold
+### x-k8s.workload.autoscale.cpuThreshold
 
-Defines the CPU utilization threshold for the horizontal pod autoscaler for the application component. See K8s [documentation](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/). This setting is only taken into account maximum number of replicas for the application component is defined.
+Defines the CPU utilisation threshold for the horizontal pod autoscaler for the application component. See K8s [documentation](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/). This setting is only taken into account maximum number of replicas for the application component is defined.
 
-### Default: `70` (70% cpu utilization)
+#### Default: `70` (70% cpu utilization)
 
-### Possible options: Arbitrary integer value. Example: `80`.
+#### Possible options: Arbitrary integer value. Example: `80`.
 
-> autoscale-cpu-threshold:
+> x-k8s.workload.autoscale.cpuThreshold:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.autoscale-cpu-threshold: 80
+    x-k8s:
+      workload:
+        autoscale:
+          maxReplicas: 10
```

Comment:
maybe add a comment here that assumes replicas to be set to a value lower than 10? Similar comment in the example below.

Reply:
I guess this hasn't been addressed? I think it'd be useful to provide some context here, especially because we present example containing more than one setting that this section is dedicated to.